waltid-services/waltid-issuer-api/src/main/kotlin/id/walt/issuer/issuance/IssuerApi.kt

```diff
@@ -449,3 +527,23 @@ fun Application.issuerApi() {
         }
     }
 }
+
+private suspend fun PipelineContext<Unit, ApplicationCall>.throwError(
+    it: Throwable
+) {
+    when (it) {
+        is BadRequestException -> context.respond(HttpStatusCode.BadRequest, it.message ?: "Bad request.")
+        is JedisConnectionException -> context.respond(
+            HttpStatusCode.InternalServerError,
+            "Distributed session management couldn't be initialized : Cannot connect to redis server."
+        )
+
+        is JedisAccessControlException -> context.respond(
+            HttpStatusCode.InternalServerError,
+            "Distributed session management couldn't be initialized : Cannot access redis server, wrong username/password."
```

Comment:
I imagine a scenario in which the user sets up the access credentials in the wrong place. For them, the credentials are correct but, as the system is looking elsewhere, they may not understand why the error occurred.

So, my first reaction would be to suggest including the username and password that was used here in the response, but I don't think that's a good idea. Any ideas on how to make it clear to the user which user/password is wrong in such a way he can quickly identify the source of the problem?

Reply:
i think its already clear with : Cannot access redis server, wrong username/password